src/README.md

```diff
@@ -0,0 +1,25 @@
+# igraph/src
+
+## Updating the C sources
+
+1. Check out and update the `cigraph` submodule to the desired version.
+    ```sh
+    git submodule init
+    git submodule update
+    # git submodule magic
```

Comment:
Was this line meant to be left in?

Reply:
We could replace it by an example to update to a specific SHA. I'm not familiar with `git submodule` syntax.

Reply:
Ah, that's what you meant. I'll complement this README file later to make that clear, but feel free to merge anytime.